stacks-signer/src/signer.rs

```diff
@@ -1453,3 +1472,41 @@ impl Signer {
         Ok(())
     }
 }
+
+fn encrypt(private_key: &Scalar, msg: &[u8]) -> Vec<u8> {
+    wsts::util::encrypt(&derive_encryption_key(private_key), msg, &mut OsRng)
```

Comment:
I'm not sure that either this method nor `decrypt()` should panic?  I think they should instead return a `Result`.  Can you describe the conditions under which they'd panic?

Reply:
Agreed. These can only fail if the underlying aes-gcm implementation fails. This seems to only fail in the current implementation if the encrypted message is larger than 1 << 36 = 68719476736. So in our case this is infallible, though it's probably safer to return an error in case the library gets updated and the underlying implementation somehow would turn fallible.

I'll change to return an error if these fail.

Reply:
Updated to return results.